src/lxc/caps.c

```diff
@@ -247,6 +244,8 @@ bool lxc_file_cap_is_set(const char *path, cap_value_t cap, cap_flag_t flag)
 	cap_is_set = lxc_cap_is_set(caps, cap, flag);
 	cap_free(caps);
 	return cap_is_set;
+	#else
+	return true;
```

Comment:
Currently all of the caps functions return `true` when they are not defined. Not sure if reversing it is worth it.